kotlin/ktor/src/Application.kt

```diff
@@ -0,0 +1,26 @@
+package ktor.benchmark
+
+import io.ktor.application.*
+import io.ktor.response.*
+import io.ktor.request.*
+import io.ktor.routing.*
+import io.ktor.http.*
+
+fun main(args: Array<String>): Unit = io.ktor.server.netty.EngineMain.main(args)
+
+@Suppress("unused") // Referenced in application.conf
+fun Application.module() {
+    routing {
+        get("/") {
+            call.respondText("", contentType = ContentType.Text.Plain, status = HttpStatusCode.OK)
+        }
+        get("/user/{id}") {
```

Comment:
Could this route be constrains ?
I mean can we for `id` to be numeric ?

Reply:
3817050